src/test/resources/de/samply/common/ldmclient/cql/measureReport.json

```diff
@@ -0,0 +1,235 @@
+{
+  "date": "2022-07-06T06:09:37.647961628Z",
+  "group": [
+    {
+      "stratifier": [
```

Comment:
Please remove the stratifier, because they are not needed. It's better to keep the test data as minimal as possible.